Sources/Build/describe().swift

```diff
@@ -24,6 +24,10 @@ public func describe(_ prefix: String, _ conf: Configuration, _ modules: [Module
         throw Error.noModules
     }
 
+    if modules.count == 1, let module = modules.first as? CModule, !(module is ClangModule) {
```

Comment:
Could we get a comment here explaining what this code is checking for and why it is necessary?